website/docs/r/integration_docker_v2.html.markdown

```diff
@@ -23,13 +23,14 @@ Supported Docker V2 registries:
 * JFrog Artifactory (On prem 7.2.1 and cloud)
 * JFrog Platform (On prem 7.2.1 and cloud)
 
-~> **Note:** You must whitelist the Lacework outbound IPs to allow the vulnerability scanner to communicate with your private registries. See [Lacework Outbound IPs](https://support.lacework.com/hc/en-us/articles/360052140433)
+~> **Note:** You must whitelist the Lacework outbound IPs to allow the vulnerability scanner to communicate with your private registries. See [Lacework Outbound IPs](https://docs.lacework.com/lacework-outbound-ips)
 
 ## Example Usage
 
 ```hcl
 resource "lacework_integration_docker_v2" "jfrog" {
   name            = "My Docker V2 Registry"
+  non_os_package_support = true
```

Comment:
Users don't need this since it is an optional field, we can skip showing it on the example usage
```suggestion
```

Reply:
Have removed from the documentation as suggested